test/keyboardNavigation.test.tsx

```diff
@@ -9,7 +9,7 @@ import { setup, getSelectedCell, validateCellPosition, getCellsAtRowIndex, getGr
 type Row = undefined;
 
 const rows: readonly Row[] = Array(100);
-const summaryRows: readonly Row[] = [undefined, undefined];
+const bottomSummaryRows: readonly Row[] = [undefined, undefined];
```

Comment:
Testing keyboard navigation with top rows might be good too, it's too easy to make off-by-one mistakes